src/cli.js

```diff
@@ -218,17 +218,33 @@ async function runCmd (argv, stdout, stderr) {
       );
       if (existsSync(outDir))
         rimraf.sync(outDir);
+      mkdirp.sync(outDir);
       run = true;
 
     // fallthrough
     case "build":
       if (args._.length > 2)
         errTooManyArguments("build");
 
+      function hasTypeModule (path) {
+        let root = resolve('/');
+        while ((path = resolve(path, '..')) !== root) {
```

Comment:
This might break on windows when `root === "C:\"` and `path === "D:\some\path"`, as it never reaches root.

You can do it this way:

```suggestion
        while (path !== (path = resolve(path, '..'))) {
```

Reply:
Nice syntax!

Reply:
Usually I would avoid = in conditions, but in this case it's really tense, while moving it out of the condition is really verbose...